docs/Writing Transactions to a Ledger for an Un-privileged Author.md

```diff
@@ -0,0 +1,460 @@
+
+The procedure can be used to write transactions to any ledger by simply initializing the containerized Indy-CLI environment with the genisis file from the desired pool.
+
+## Prerequisites
+
+The following process uses the Indy-CLI container from [von-network](https://github.com/bcgov/von-network) which provides a containerized `indy-cli` environment and facilities to mount a volume containing `indy-cli` batch script templates and perform variable substitution on those templates, turning them into reusable scripts.
+
+Build and start [von-network](https://github.com/bcgov/von-network); `./manage build`, `./manage start`.
+
+Register DIDs (using seeds) for the following purposes using the `von-network` interface `http://localhost:9000`:
+- TOB
+  - Seed: the_org_book_0000000000000000000
+  - DID: Leave blank
+  - Alias: the-org-book
+  - Role: None
+
+  - Resulting Registration:
+    - DID: 25GuF6zjiywpU1iF4kNJPQ
+    - Verkey: awcJaDCU36RQxeqMeRKcrVHvXRZU3Ysbj6sCf6W7Q6A
+
+- Agent
+  - Seed: 0000000000000000000000000MyAgent
+  - DID: Leave blank
+  - Alias: my-agent
+  - Role: None
+
+  - Resulting Registration:
+    - DID: NFP8kaWvCupbDQHQhErwXb
+    - Verkey: Cah1iVzdB6UF5HVCJ2ENUrqzAKQsoWgiWyUopcmN3WHd
+
+- Endorser
+  - Seed: ENDORSER123450000000000000000000
+  - DID: Leave blank
+  - Alias: my-endorser
+  - Role: Endorser
+
+  - Resulting Registration:
+    - DID: DFuDqCYpeDNXLuc3MKooX3
+    - Verkey: 7gTxoFyCpMGfxuwBNXYn1icuyh8DqDpRGi12hj5S2pHk
+
+Reset your Indy-CLI container's environment:
+```
+./manage cli reset
+```
+
+Initialize the pool for your Indy-CLI container's environment:
+```
+./manage cli init-pool localpool http://192.168.65.3:9000/genesis
+```
+
+## Start up the applications
+
+
+### 1. Run OrgBook Ledger connection parameters (assuming you have already done the `./manage build` to initialize):
+```
+seed=the_org_book_0000000000000000000 AUTO_REGISTER_DID=0 LEDGER_URL=not_used GENESIS_URL=http://192.168.65.3:9000/genesis ./manage start
+```
+
+### 2. Run von-agent-template (assuming you have already done the `. init.sh` and `./manage build` to initialize):
+
+```
+INDY_LEDGER_URL="" AUTO_REGISTER_DID=0 INDY_GENESIS_URL=http://192.168.65.3:9000/genesis WALLET_SEED_VONX=0000000000000000000000000MyAgent ./manage start
+```
+
+**Note:** The DID for the application must already exist on the ledger.
+
+**Note:** The agent will fail to fully syncronize since it's DID doesn't have privileges to write the schema(s) and cred-defs to the ledger.
+
+**Note:** Leave the OrgBook and VON Agent services running, we will be using the agent's wallet database.
+
+## Create and Write the Schema and Cred Def to the Ledger
+
+_In the following examples:_
+- Commands are run in your `von-network` directory, as we are using the Indy-CLI container.
+- `/c/von-network/cli-scripts` is the abosulte path to the scripts contained in this repository.
```

Comment:
If I understand correctly, it appears that the user will need to replace the absolute path to the scripts folder in order to execute the commands. Would it be possible/beneficial to auto-determine the path, or use an environment variable instead of typing it in the command string?